src/utils/transactionUtils/transactionUtils.ts

```diff
@@ -380,9 +390,9 @@ export function getInteractionTypeFromField(field: string) {
 
 export function mapTransactionDataKeyToPayload(
   interactionType: ValidInteractionType,
-  data: string | number | Array<string | number>,
+  data: string | number | Array<string | number | PDNTContractJSON>,
```

Comment:
i think we should just change this to an object vs. an array, but can be done in another PR.

Reply:
you may be right.... i was originally using this function for url params were all the data was stored in the url under the `data` search params, and was sticking with that logic here.